boards/arc/emsdp/doc/index.rst

```diff
@@ -44,6 +44,9 @@ The following table shows the hardware features supported for different core con
 | Secure    | N   | N   | N    | N    | Y        | N    | N     |
 +-----------+-----+-----+------+------+----------+------+-------+
 
+Support two 32 MByte Quad-SPI Flash memory, one is for configuration and operation, the other
+is user SPI-FLASH, its sample can be found in ``samples/drivers/spi_flash``.
+
```

Comment:
The 'west flash' section below may be confusing now, maybe add a remark here that execution from flash and 'west flash' is not yet supported?

Reply:
And while you're improving this doc, maybe add other peripherals that are supported here as well (like the EMSK table and other boards have).